src/imageLoader/wadors/combineFrameInstance.js

```diff
@@ -0,0 +1,93 @@
+function getTagValue(tag, justElement = true) {
```

Comment:
I'd suggest separating getTagValue out into it's own file so it can be re-used.

Reply:
Done